src/test/testFiles/stylesheet/variableOverrideStylesheet.json

```diff
@@ -1,14 +1,18 @@
 {
-  "stack1": {
-    "textColor": "BLUE",
-    "borderColor": "ORANGE"
+  "variables": {
+    "stack1": {
+      "textColor": "BLUE",
+      "borderColor": "ORANGE"
+    }
   },
-  "Stack": {
-    "borderColor": "YELLOW",
-    "textColor": "GREEN",
-    "animate": {
-      "borderColor": "RED",
-      "textColor": "RED"
+  "dataStructures": {
```

Comment:
This makes it less like CSS which is what we were trying to make at as close to so it's relatable. Could this not be a map like before?

Reply:
Is it because of the code tracking key?

Reply:
Not really as the Gson needs an object type to map to. And code tracking is just a string which doesn't need color etc

Reply:
I would say this makes the style sheet easier to understand for the user. Sectioning it into variables, types etc. Will mean its easier for them to read and easier for us to extend

Reply:
I will open another PR for that

Reply:
Can this be approved then?

Reply:
Added another comment